pkg/kubefed2/unjoin.go

```diff
@@ -180,15 +190,18 @@ func UnjoinCluster(hostConfig, clusterConfig *rest.Config, federationNamespace,
 	}
 
 	if removeFromRegistry {
-		removeFromClusterRegistry(hostConfig, clusterNamespace, clusterConfig.Host, unjoiningClusterName, dryRun)
+		removeFromClusterRegistry(hostConfig, clusterNamespace, unjoiningClusterName, dryRun)
 	}
 
-	deletionSucceeded := deleteRBACResources(clusterClientset, federationNamespace, unjoiningClusterName, hostClusterName, dryRun)
+	var deletionSucceeded bool = false
```

Comment:
The default value for bool is false, so assigning false is unnecessary.  Both of the following forms are preferred:

```golang
// shorthand-style, useful for initializing non-pointer types  
deletionSucceeded := false
// var-style, with default to zero-value for type (false for bool)
var deletionSucceeded bool
```